### PR TITLE
feat: update bundle-loader endpoint to v3

### PR DESCRIPTION
**Pull request by gorgias-botond-dombi**
**Changes**:
- Update bundle-loader to v3
- Change display name for app id parameter

### CODE REVIEW

**Comment by gorgias-botond-dombi** [comment]
@TomyCesaille Should this change have backward compatibility and support loading the bundle still if the applicationId is a number?

**Comment by TomyCesaille** [comment]
> @TomyCesaille Should this change have backward compatibility and support loading the bundle still if the applicationId is a number?

Ah! Excellent question. If someone updates the tags in GTM without looking at the Gorgias page... This person might upgrade its applicationId number to v3.... that's bad because this endpoints expects the appKey 🙈
In that case, I think the error returned in the browser is clear enough for support lvl1 to kick-in and solve the problem with the customer (change to appKey). Looks like the best option to me because 1. we don't do backward compatibility 2. we help merchants to migrate in v3 for real.

File: template.tpl [hunk 1]
```diff
@@ -33,7 +33,7 @@ ___TEMPLATE_PARAMETERS___
   {
     "type": "TEXT",
     "name": "gorgiasChatAppID",
```
Comment (TomyCesaille) [inline template.tpl]: Not renaming this? I wonder if it 'helps' merchants to detect that the 'contract' has changed with this new tag?

**Review by TomyCesaille — COMMENTED** [review]
I think there is another description file to update (version at least)

**Comment by juancarlosqr** [comment]
> I think there is another description file to update (version at least)

File: template.tpl (hunk 1 above)
Comment (gorgias-botond-dombi) [reply template.tpl]: I'm renaming it, it's a breaking change anyway

**Comment by TomyCesaille** [comment]
Also, make sure that this endpoint works :D (with the `GTMContainerId` parameter, since this part of the code was dormant for a year or so)

**Comment by gorgias-botond-dombi** [comment]
@TomyCesaille I believe the metadata.yaml file will have to be updated after this PR is merged? Since we will get a new commit sha? (Looking at past commits)

**Comment by gorgias-botond-dombi** [comment]
@TomyCesaille The endpoints supports the `GTMContainerId` query param

**Comment by TomyCesaille** [comment]
> I believe the metadata.yaml file will have to be updated after this PR is merged? Since we will get a new commit sha? (Looking at past commits)

Ah maybe! I haven't looked at the whole process to submit the new tag version. I remember something around sharing the github repo to google or something like this 🤔

**Comment by juancarlosqr** [comment]
>  I believe the metadata.yaml file will have to be updated after this PR is merged? Since we will get a new commit sha?

@gorgias-botond-dombi yes, we need to use the sha commit from this PR, and could you please also update the doc link in there for [this one](https://docs.gorgias.com/en-US/chat-getting-started-81789#setup-instructions-for-google-tag-manager)? 

Thanks

File: metadata.yaml [hunk 2]
```diff
@@ -1,9 +1,11 @@
 homepage: "https://www.gorgias.com/"
-documentation: "https://docs.gorgias.com/gorgias-chat/chat-getting-started#setup_instructions_for_google_tag_manager"
+documentation: "https://docs.gorgias.com/en-US/chat-getting-started-81789#setup-instructions-for-google-tag-manager"
 versions:
   # Latest version
+  - sha: c92a48c53b05bee9225a07cac10da7a9cecd1572
+    changeNotes: Introduce the v3 snippet. Use app key instead of app id.
```
Comment (juancarlosqr) [inline metadata.yaml]: I don't think this is going to work, once you merge is going to be a new sha

**Review by juancarlosqr — DISMISSED** [review]

**Comment by gorgias-botond-dombi** [comment]
I've pushed another commit to update the `metadata.yaml` @juancarlosqr included the doc update too

File: metadata.yaml (hunk 2 above)
Comment (gorgias-botond-dombi) [reply metadata.yaml]: Let's remove it from the PR if it looks good for you.

**Review by juancarlosqr — APPROVED** [review]

**Pull request merged**